nsv13/code/game/machinery/computer/tactical.dm

```diff
@@ -78,6 +78,14 @@
 	data["flakrange"] = linked.get_flak_range(linked.last_target)
 	data["integrity"] = linked.obj_integrity
 	data["max_integrity"] = linked.max_integrity
+	if(istype(linked, /obj/structure/overmap/small_craft/transport))
```

Comment:
Should these be for all small craft instead of just transport?

Reply:
Oh true it should

Reply:
Made the change.